File: sdkwork-app-sdk-typescript/src/types/algorithm-config-vo.ts
/** 算法配置响应 */
export interface AlgorithmConfigVO {
  /** 创建时间 */
  createdAt?: string;
  /** 更新时间 */
  updatedAt?: string;
  /** 算法类型 */
  algorithmType?: string;
  /** 算法参数 */
  algorithmParams?: unknown;
  /** 配置版本 */
  version?: string;
}

File: sdkwork-app-sdk-java/src/main/java/com/sdkwork/app/model/AlgorithmConfigVO.java
package com.sdkwork.app.model;

public class AlgorithmConfigVO {
    private String createdAt;
    private String updatedAt;
    private String algorithmType;
    private Object algorithmParams;
    private String version;

    public String getCreatedAt() {
        return this.createdAt;
    }
    
    public void setCreatedAt(String createdAt) {
        this.createdAt = createdAt;
    }

    public String getUpdatedAt() {
        return this.updatedAt;
    }
    
    public void setUpdatedAt(String updatedAt) {
        this.updatedAt = updatedAt;
    }

    public String getAlgorithmType() {
        return this.algorithmType;
    }
    
    public void setAlgorithmType(String algorithmType) {
        this.algorithmType = algorithmType;
    }

    public Object getAlgorithmParams() {
        return this.algorithmParams;
    }
    
    public void setAlgorithmParams(Object algorithmParams) {
        this.algorithmParams = algorithmParams;
    }

    public String getVersion() {
        return this.version;
    }
    
    public void setVersion(String version) {
        this.version = version;
    }
}
